tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -62,12 +74,12 @@ function configure_cri_runtime() {
 	crio)
 		configure_crio
 		;;
-	containerd | k3s | k3s-agent)
+	containerd | k3s*)
```

Comment:
This change is good, but it's unrelated to what we're trying to achieve, so I'd have it in a different commit (still part of the same PR, but in a different commit).

Reply:
I actually need this to get it worked in my env. which I install k3s with install.sh script. The script names k3s service as k3s_service.  I have not verified how k3s look like if it is installed in another way, like using package manger, so I do it in this way to avoid breaking early implementation, which should be tested and works OK.

Reply:
Could you add this as a previous commit, explaining this in the commit message?

This is really good info and deserves its own commit.

Reply:
see https://github.com/kata-containers/kata-containers/pull/4065

Reply:
Thanks for splitting the commit.  I have closed #4065 as having the commit split but part of this very same PR is preferred.